### DISCUSSION TITLE
how to write the code of POST request with json body in jython ？

### DISCUSSION THREAD

**Discussion by hqsh** [naver/ngrinder#1026] [Q&A]
json body may be: {"a": 1, "b": ["x", "y"], "c": {"c1": 1, "c2": 2}}

**Comment by imbyungjun** [comment]
![image](https://github.com/user-attachments/assets/a99d041c-936d-42af-a888-f72ea61e9c9e)
Try advanced configuration like this when you creating a new Jython script.
This will create a new Jython script with JSON handling template.
